datalad_container/adapters/docker.py

```diff
@@ -44,6 +48,9 @@ def save(image, path):
     # Use a temporary file because docker save (or actually tar underneath)
     # complains that stdout needs to be redirected if we use Popen and PIPE.
     with tempfile.NamedTemporaryFile() as stream:
+        if on_windows:
```

Comment:
do we really need to condition it with `on_windows`? sounds like it could just be generally done, or am I missing something?

Reply:
Oh, true. For some reason, I didn't think of this. Will adjust.